### PR TITLE
Cet 3115 add scorecard display options

### PR DESCRIPTION
**Pull request by cristinabuenahora**
Adds option to add badges to Scorecards in list view

### CODE REVIEW

File: src/extensionApi.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export type CortexYaml = {
     url: string;
     description?: string;
   }[];
-  'x-cortex-service-groups'?: string[];
+  'x-cortex-groups'?: string[];
```
Comment (FoodProduct) [inline src/extensionApi.ts:35]: Isn't this a breaking API change?

Comment (cristinabuenahora) [reply src/extensionApi.ts:35]: I think this was old, we actually have x-cortex-groups now

**Review by FoodProduct — APPROVED** [review]

File: src/extensionApi.ts (hunk 1 above)
Comment (nikhilunni) [reply src/extensionApi.ts:35]: the extensions maps to a `Partial<CortexYaml>` so i think technically not a breaking change?

**Review by nikhilunni — APPROVED** [review]

File: src/extensionApi.ts (hunk 1 above)
Comment (cristinabuenahora) [reply src/extensionApi.ts:35]: Let me revert just in case

**Pull request merged**